crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -191,134 +200,6 @@ impl Display for CheckpointSummary {
-        self.summary.sequence_number
-    }
-
-    pub fn content_digest(&self) -> CheckpointContentsDigest {
-        self.summary.content_digest
-    }
-
-    pub fn previous_digest(&self) -> Option<CheckpointDigest> {
-        self.summary.previous_digest
-    }
-
-    pub fn next_epoch_committee(&self) -> Option<&[(AuthorityName, StakeUnit)]> {
-        self.summary
-            .end_of_epoch_data
-            .as_ref()
-            .map(|e| e.next_epoch_committee.as_slice())
-    }
-}
-
-impl<S: Debug> Display for CheckpointSummaryEnvelope<S> {
-    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        writeln!(f, "{}", self.summary)?;
-        writeln!(f, "Signature: {:?}", self.auth_signature)?;
-        Ok(())
-    }
-}
-
-pub type SignedCheckpointSummary = CheckpointSummaryEnvelope<AuthoritySignInfo>;
-
-impl SignedCheckpointSummary {
-    /// Create a new signed checkpoint proposal for this authority
-    pub fn new(
-        epoch: EpochId,
-        sequence_number: CheckpointSequenceNumber,
-        network_total_transactions: u64,
-        authority: AuthorityName,
-        signer: &dyn Signer<AuthoritySignature>,
-        transactions: &CheckpointContents,
-        previous_digest: Option<CheckpointDigest>,
-        epoch_rolling_gas_cost_summary: GasCostSummary,
-        end_of_epoch_data: Option<EndOfEpochData>,
-        timestamp_ms: CheckpointTimestamp,
-    ) -> SignedCheckpointSummary {
-        let checkpoint = CheckpointSummary::new(
-            epoch,
-            sequence_number,
-            network_total_transactions,
-            transactions,
-            previous_digest,
-            epoch_rolling_gas_cost_summary,
-            end_of_epoch_data,
-            timestamp_ms,
-        );
-        SignedCheckpointSummary::new_from_summary(checkpoint, authority, signer)
-    }
-
-    pub fn new_from_summary(
-        checkpoint: CheckpointSummary,
-        authority: AuthorityName,
-        signer: &dyn Signer<AuthoritySignature>,
-    ) -> SignedCheckpointSummary {
-        let epoch = checkpoint.epoch;
-        let auth_signature = AuthoritySignInfo::new(
-            epoch,
-            &checkpoint,
-            Intent::default().with_scope(IntentScope::CheckpointSummary),
-            authority,
-            signer,
-        );
-        SignedCheckpointSummary {
-            summary: checkpoint,
-            auth_signature,
-        }
-    }
-
-    pub fn authority(&self) -> &AuthorityName {
-        &self.auth_signature.authority
-    }
-
-    /// Checks that the signature on the digest is correct, and verify the contents as well if
-    /// provided.
-    pub fn verify(
-        &self,
-        committee: &Committee,
-        contents: Option<&CheckpointContents>,
-    ) -> Result<(), SuiError> {
-        fp_ensure!(
-            self.summary.epoch == committee.epoch,
-            SuiError::from("Epoch in the summary doesn't match with the signature")
-        );
```

Comment:
do we need this check still?

Reply:
I'm not sure if its really necessary, but I will add it back just in case